### ISSUE TITLE
docker should be `gcloud docker`

### ISSUE THREAD

**Issue by machinshin** [circleci/docker-hello-google#7]
in addition, the current versions of gcloud seperate gcloud args from docker args with `--`

so should be `gcloud docker -- build ... -t ..etc..`

**Comment by deesx** [comment]
they also do it now for push (`gcloud docker -- push etc…`)

**Comment by beast** [comment]
Yeah, docker build is for local image, gcloud docker is only for container registry push.   https://cloud.google.com/container-registry/docs/quickstart
